ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/ExceptionStatus.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.shell;
+
+/**
+ * Exception status related to invoker.
+ *
+ * @since 2.0.0
+ */
+public enum ExceptionStatus {
+    SUCCESS("success"),
+    TREE_PARSER_FAILED("tree_parser_failed"),
+    SNIPPET_FAILED("snippet_failed"),
+    INVOKER_FAILED("invoker_failed");
+
+    String status;
```

Comment:
Better make the field private

Reply:
Updated in 9f270db8b7259fb63e77545e8454623db4d6c3a7